paddle/framework/block_desc.cc

```diff
@@ -34,6 +34,11 @@ VarDescBind *BlockDescBind::Var(const std::string &name) const {
   return it->second.get();
 }
 
+bool BlockDescBind::HasVar(const std::string &name) const {
+  auto it = vars_.find(name);
```

Comment:
return vars_.count(name) != 0;

Reply:
`find` is better to describe the semantic of `has`.  
> Since a map can only have at most one key, count will essentially stop after one element has been found. However, in view of more general containers such as multimaps and multisets, find is strictly better if you only care whether some element with this key exists, since it can really stop once the first matching element has been found.

For `map`,  the implement of `Has` is just:
```cpp
size_type count(const key_type& __x) const                                                       
    { return _M_t.find(__x) == _M_t.end() ? 0 : 1; 
}  
```

https://stackoverflow.com/questions/25490357/checking-for-existence-in-stdmap-count-vs-find

Reply:
I see. `find` could be more efficient than `count`. So here maybe `return vars_.find(name) != vars_end();`?

Reply:
yes, have change to this

Reply:
done